ext/standard/basic_functions.stub.php

```diff
@@ -1487,7 +1488,7 @@ function version_compare(string $version1, string $version2, ?string $operator =
 /* win32/codepage.c */
 
 #ifdef PHP_WIN32
-function sapi_windows_cp_set(int $cp): bool {}
+function sapi_windows_cp_set(int $codepage): bool {}
```

Comment:
This seems to be written as two words (just like codepoint above).  I guess they could stay as-is, if you prefer writing them as a single word.

Reply:
We are using `$codepoint` as one word everywhere though (49 times).

Reply:
ok, let's go with `$codepoint` then